### ISSUE TITLE
No jump handler was able to find this symbol (elixir mode + lsp mode)

### ISSUE THREAD

**Issue by kpanic** [syl20bnr/spacemacs#12765]
#### Description :octocat:
No jump handler was able to find this symbol (elixir mode)

#### Reproduction guide :beetle:
- Start Emacs
- On an Elixir file, do `, g g` in a function

*Observed behaviour:* :eyes: :broken_heart:
The message **No jump handler was able to find this symbol** is displayed
although calling `M-x lsp-find-definition` works.

*Expected behaviour:* :heart: :smile:
`, g g` should work :)

#### System Info :computer:
- OS: gnu/linux
- Emacs: 26.3
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 37e326e1c)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(yaml javascript auto-completion better-defaults
      (colors :variables colors-enable-nyan-cat-progress-bar t)
      osx dash deft python elixir lsp emacs-lisp erlang git github html emoji erc markdown shell spell-checking org
      (shell :variables shell-default-height 30 shell-default-position 'bottom)
      search-engine syntax-checking themes-megapack version-control)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS GSETTINGS GLIB NOTIFY LIBSELINUX GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM MODULES THREADS XWIDGETS LIBSYSTEMD LCMS2

**Comment by mpanarin** [comment]
please provide something to reproduce on. As, for example, for me it works fine.

**Comment by mpanarin** [comment]
Also, are you sure you even use lsp in your configuration?
As I don't see you setting lsp as a backend for elixir. By default it is alchemist
i.e.:
```elisp
     (elixir :variables
             elixir-backend 'lsp)
```

**Comment by kpanic** [comment]
I use:

```elisp
(use-package lsp-mode
    :commands lsp
    :ensure t
    :diminish lsp-mode
    :hook
    (elixir-mode . lsp)
    :init
    (add-to-list 'exec-path "/my/path/to/elixir-ls/release"))
```

in `dotspacemacs/user-config`

**Comment by kpanic** [comment]
@mpanarin my error is exactly this one: https://elixirforum.com/t/emacs-elixir-setup-configuration-wiki/19196/99

What I did:

* `mix deps.update --all` with https://github.com/JakeBecker/elixir-ls from the lates `master`
* `mix compile`
* `mix elixir_ls.release`

And it looks like it's working.

Although I have to test it better. but looks like an elixir sense issue that was solved in a more recent release. 
And https://github.com/JakeBecker/elixir-ls looks is behind regarding the deps

I will keep open the issue until I will experiment a little bit more
